src/Agent.Worker/Release/Artifacts/FileContainer/FileContainerItemCache.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.TeamFoundation.Core.WebApi;
+using Microsoft.VisualStudio.Services.Agent.Worker;
+using Microsoft.VisualStudio.Services.FileContainer;
+using Microsoft.VisualStudio.Services.FileContainer.Client;
+using Microsoft.VisualStudio.Services.WebApi;
+
+namespace Agent.Worker.Release.Artifacts.FileContainer
+{
+    public class FileContainerItemCache
+    {
+        public FileContainerItemCache(Uri teamProjectCollectionUrl, long containerId, string projectName, string accessToken, string rootItemPath = null, IExecutionContext logger = null, bool includeDownloadTickets = false)
```

Comment:
I don't understand why I should have to go read a blog post to understand code that isn't required.